src/main/java/io/jenkins/plugins/pipelinegraphview/cards/items/UserIdCauseRunDetailsItem.java

```diff
@@ -26,8 +27,9 @@ public static Optional<RunDetailsItem> get(WorkflowRun run) {
                 .map(userIdCause -> (Cause.UserIdCause) userIdCause)
                 .map(userIdCause -> User.get(userIdCause.getUserId(), false, new HashMap<>()))
                 .filter(Objects::nonNull)
-                .map(user -> ItemContent.of(Messages.cause_user(user.getDisplayName())))
-                .<RunDetailsItem>map(content -> new RunDetailsItem.RunDetail(new Ionicon("person-outline"), content))
+                .<RunDetailsItem>map(user -> new RunDetailsItem.RunDetail(
+                        new SimpleIcon(UserAvatarResolver.resolve(user, "48x48")),
```

Comment:
Rather than using a SimpleIcon, to give it some meaning what are your thoughts on having a UserIcon class?

```java
final class UserIcon implements Icon {

    private final String avatar;

    public UserIcon(User user) {
        avatar = UserAvatarResolver.resolve(user, "48x48");
    }

    @Override
    public String value() {
        return avatar;
    }
}
```

Reply:
I agree - would make the change if we had more than the one implementation/this will be killed eventually once we start using cores Detail API.